knockapi/resources/objects.py

```diff
@@ -453,8 +453,8 @@ def list_subscriptions(self, collection, id, options={}):
 
     def get_subscriptions(self, collection, id, options={}):
         """
-        Returns subscriptions for an object as a recipient
-
+        Returns all of the active subscriptions that the specified object is subscribed to (not the subscribers of the object)
```

Comment:
```suggestion
        Returns all of the active subscriptions for which the specified object is a recipient
```
I think maybe with this word-smithing, the "not the subscribers..." part could be left out? But no strong feelings about that part.